### PR TITLE
feat: Overview proposal

### PR DESCRIPTION
**Pull request by eva-cosma**

### CODE REVIEW

File: docs/overview/container-basics.md [hunk 1]
```diff
@@ -8,8 +8,66 @@ sidebar_position: 2
...
+
+**Containers**, compared to VMs, are friendlier with the host operating system, sharing resources in
+a more streamlined manner. They don't require you to run an entire operating system, just the
+underlying libraries necessary for your application to run, and your application code. The
+industry-standard application to manage containers is [Docker](https://www.docker.com/).
+
+<center> ![img](./containerization.drawio.png) </center>
+
+Another neat advantage of containers, due to their lightweight nature they start extremely fast.
+This is extremely helpful when managin widely popular applications due to their ability to create
```
Comment (cristian-paris) [inline docs/overview/container-basics.md]:
```suggestion
This is extremely helpful when managing widely popular applications due to their ability to create
```

File: docs/overview/introduction.md [hunk 2]
```diff
@@ -24,29 +24,37 @@ sidebar_position: 1
...
-eliminates the problem of running an application on multiple systems, each of them with different
-versions of dependencies. The classic *works on my system* problem.
+ Orchestration is managing and coordinating multiple containers in a distributed environment. This
+ includes tasks such as scaling containers up or down, load balancing traffic between containers,
+ and ensuring high availability of the application. Kubernetes is a popular orchestration tool that
+ helps automate these tasks, making it easier to manage containerized applications at scale.
 
-### What does it mean to orchestrate some containers?
+ By the end of this course, you will have the skills and knowledge needed to deploy and manage
+ containerized applications in a production environment.
```
Comment (cristian-paris) [inline docs/overview/introduction.md]: Remove extra whitespace at the beginning of each line

Comment (cristian-paris) [reply docs/overview/introduction.md]: Maybe also justify the content if it looks right (i.e. the space between words does not get extremely stretched)

Comment (eva-cosma) [reply docs/overview/introduction.md]: I'm not actually sure how to do that? I thought it was due to the word-wrapping, but even after putting it all in one line it still puts a lot of space between words

**Pull request merged**